easybuild/tools/options.py

```diff
@@ -524,6 +524,8 @@ def override_options(self):
             'rpath-filter': ("List of regex patterns to use for filtering out RPATH paths", 'strlist', 'store', None),
             'rpath-override-dirs': ("Path(s) to be prepended when linking with RPATH (string, colon-separated)",
                                     None, 'store', None),
+            'rpath-wrappers-dir': ("Absolute path to directory to use for RPATH wrappers creation/use",
```

Comment:
Rather than adding an EasyBuild configuration option for this, we can just only give control over through an easyblock which can set `self.rpath_wrappers_dir` in the constructor, so there's no control from the outside to tweak the location of the generaed RPATH wrapper scripts.